x-pack/plugins/enterprise_search/public/applications/app_search/components/search_ui/components/search_ui_form.test.tsx

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+jest.mock('../utils', () => ({
+  generatePreviwUrl: jest.fn(),
+}));
+
+import { setMockValues, setMockActions } from '../../../../__mocks__';
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { ActiveField } from '../types';
+import { generatePreviwUrl } from '../utils';
+
+import { SearchUIForm } from './search_ui_form';
+
+describe('SearchUIForm', () => {
+  const values = {
+    validFields: ['title', 'url', 'category', 'size'],
+    validSortFields: ['title', 'url', 'category', 'size'],
+    validFacetFields: ['title', 'url', 'category', 'size'],
+    titleField: 'title',
+    urlField: 'url',
+    facetFields: ['cateegory'],
```

Comment:
not blocking, smol typo here
```suggestion
    facetFields: ['category'],
```